cmd/control/util.go

```diff
@@ -3,13 +3,15 @@ package control
 import (
 	"bufio"
 	"fmt"
+	"os"
 	"strings"
 
 	log "github.com/Sirupsen/logrus"
 )
 
-func yes(in *bufio.Reader, question string) bool {
+func yes(question string) bool {
 	fmt.Printf("%s [y/N]: ", question)
+	in := bufio.NewReader(os.Stdin)
```

Comment:
I am expecting that creating a new reader isn't going to modify the underlying stream - only a read / write / seek would do that - and we're only reading to the next `\n`

but - yeah, I'll go read the code and test a little to make sure my assumptions are sane :)